<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMusic</class>
 <widget class="QDialog" name="NewMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ajouter une musique</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditPath">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelPath">
      <property name="text">
       <string>Chemin: </string>
      </property>
      <property name="buddy">
       <cstring>lineEditPath</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditTitle"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Titre: </string>
      </property>
      <property name="buddy">
       <cstring>lineEditTitle</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelArtist">
      <property name="text">
       <string>Artiste: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="selectArtist"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonSubmit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>NewMusic</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
